TipTac/ttCore.lua

```diff
@@ -2811,7 +2811,7 @@ function tt:GetAnchorPosition(tip)
 		end
 	end
 	
-	local mouseFocus = GetMouseFocus();
+	local mouseFocus = GetMouseFoci();

```

Comment:
GetMouseFoci() returns a table.  I'm not sure this is right.  Probably should be something like:

```
local mouseFocus
if GetMouseFocus then
  mouseFocus = GetMouseFocus()
elseif GetMouseFoci then
  local foci = GetMouseFoci()
  mouseFocus = foci[1] or nil
end
```

but that might be better as a function in LibFroznFunctions:

```
function LibFroznFunctions:GetMouseFocus()
    if GetMouseFocus then
        return GetMouseFocus()
    elseif GetMouseFoci then
        local foci = GetMouseFoci()
        return foci[1] or nil
    end

    return nil
end
```

and then called like:

```
local mouseFocus = LibFroznFunctions:GetMouseFocus()
```